application/build.gradle

```diff
@@ -7,7 +7,7 @@ plugins {
 description 'Corda Application'
 
 dependencies {
-    api 'net.corda.kotlin:kotlin-stdlib-jdk8-osgi'
+    api 'org.jetbrains.kotlin:kotlin-osgi-bundle'
```

Comment:
Does that mean there is now an official kotlin OSGi bundle? That would be great!

Reply:
> Does that mean there is now an official kotlin OSGi bundle? That would be great!

There was _always_ an official Kotlin OSGi bundle, and we used it. The problem was that it previously targeted Java 6 and so did not include the contents of the `kotlin-stdlib-jdk7` and `kotlin-stdlib-jdk8` artifacts. However, Kotlin 1.8.x targets Java 8 and so the official OSGi bundle is now _finally_ complete.